Week1/homework/app.js

```diff
@@ -1,11 +1,138 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // Q1.1
+
+  const books = [
+    'animal_farm',
+    'veronica_decide_to_die',
+    '1984',
+    'the_lord_of_the_rings',
+    'the_book_thief',
+    'romeo_and_juliet',
+    'the_color_purple',
+    'alice_in_wonderland',
+    'frankenstein',
+    'the_kite_runner',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // Q1.3
+
+  function booksGenerator() {
+    const bookList = document.createElement('ul');
+    const main = document.getElementById('main');
+    main.appendChild(bookList);
+    for (let i = 0; i < books.length; i++) {
+      const Li = document.createElement('li');
```

Comment:
Be careful with naming certain variables with a capital letter. There are conventions about this: [https://www.w3schools.com/js/js_conventions.asp](url).